apps/google-analytics-4/lambda/src/app.ts

```diff
@@ -8,21 +9,46 @@ import {
   GoogleApiServerError,
 } from './services/google-api';
 import { ServiceAccountKeyFile } from './types';
+import { verifySignedRequestMiddleware } from './lib/verify-signed-request-middleware';
+import { InvalidSignature } from './lib/errors/invalid-signature';
+import { UnableToVerifyRequest } from './lib/errors/unable-to-verify-request';
+
+dotenv.config(); // TODO: load env vars from .env.local
+
 const app = express();
 
+// allow all OPTIONS requests
+app.options('/*', function (req, res, _next) {
+  res.header('Access-Control-Allow-Origin', '*');
+  res.header('Access-Control-Allow-Methods', 'GET,PUT,POST,DELETE,OPTIONS');
+  res.header(
+    'Access-Control-Allow-Headers',
+    'Content-Type, Authorization, Content-Length, X-Requested-With'
```

Comment:
Just noticing this on my second pass here but I'm not sure the value we're passing here is correct for us. See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Headers

Specifically:

* I believe we'd need to whitelist explicitly the headers we do want to accept (e.g. X-Environment-Id etc.)
* Conent-Type is not needed here since it's auto safelisted https://developer.mozilla.org/en-US/docs/Glossary/CORS-safelisted_request_header

Reply:
ah, good catch! this was definitely not complete and I totally overlooked this, too

Reply:
- [] fixme